python/nav/django/settings.py

```diff
@@ -259,13 +259,13 @@
 # * https://docs.djangoproject.com/en/3.2/ref/clickjacking/
 # Example conf:
 #  [security]
-#  ssl = on
+#  needs_ssl = yes
```

Comment:
What is this an example of, really?

Reply:
How to set the config? I like docs.

Reply:
Set what config?  This does not look like valid content for `settings.py`. It also does not match up with the new option added in `webfront.conf`.  And if this is an example of what `webfront.conf` looks like, the example would be better in that file.

Reply:
If you reacted to the typo you could have flagged the typo. The typo has been fixed. I'm not a mind-reader. The actual example is also in webfront.conf.

The links are not because most of the things in those links should NOT be toggleable from webfront.conf or any other file. They're set and forget. This is the first branch of many, each adding a bit more from those links into the django settings file and hopefully *not* to the webfront.conf-file.

Eventually, we might want to have security-docs that are not a howto but a mapping from how NAV does things to how Django does things, in what Diataxis calls a reference doc. The links can move there then.

Reply:
Also, GitHub seems to be on acid, it keeps showing me the old code even though you've pushed new commits :P

Reply:
I've actually added a link to webfront.conf because that's a great idea, we should do more of that, it'll make settings.py easier to understand.